src/coreclr/System.Private.CoreLib/src/System/Threading/ThreadPool.CoreCLR.cs

```diff
@@ -234,6 +234,10 @@ private static unsafe bool InitializeConfigAndDetermineUsePortableThreadPool()
                 ? AppContextConfigHelper.GetBooleanConfig("System.Threading.ThreadPool.EnableWorkerTracking", false)
                 : GetEnableWorkerTrackingNative();
 
+#if FEATURE_OBJCWRAPPERS
```

Comment:
Is it necessary to wrap this with ifdef?

Reply:
Since the member is only used when `FEATURE_OBJCWRAPPERS` is defined an would be a no-op otherwise, I prefer having it only exist when the feature flag is defined.

Also, since technically the feature flag != an os check, I don't want to put it in a separate file that's named based on OS instead of using the feature flag.

For example, we use the feature flag for the PNSE implementation of the System.Runtime.InteropServices.ObjectiveC types that wrap this, as well as for the same case for the ComWrappers-related types that follow a similar design.

Reply:
What is the thinking of how this will be enabled? Are we going to build multiple macos coreclr frameworks?